generate/canoto.go

```diff
@@ -291,9 +294,32 @@ ${marshal}	return w
 		"storeSuffix":         storeSuffix,
 		"oneOfCacheAccessors": makeOneOfCacheAccessors(m),
 		"marshal":             makeMarshal(m),
+		"zero":                makeZeroVarName(m, generics),
 	})
 }
 
+func makeZeroVarName(m message, generics string) string {
+	staticTypes := []canotoType{
```

Comment:
```suggestion
	zeroIsUnusedTypes := []canotoType{
```
Would probably be the most verbose...